### DISCUSSION TITLE
tasmota used as Venetian Blind control, but how to set ShutterPosition to Domoticz? [solved]

### DISCUSSION THREAD

**Discussion by smitec11** [arendst/Tasmota#11540] [Q&A] [answered]
Hi all,

I am using a Wemos D1 mini with  drv8825 and 28BYJ-48 Steppermotor , standard from the Tasmota documents page.
I can not figure out how to get the ShutterPosition into Domoticz.
Has anybody got that working?
Can the slider from a "Venetian Blinds" in Domoticz be used to display and change the shutterpostion ?

(the info for the domoticz configuration in tasmota is not very clear to me :-( )
regards,
Chris

**Comment by TrDA-hab** [comment, marked as answer]
I've never worked for Domoticz, only MajorDoMo.
I configured for myself (receive and send values  ShutterPosition via MQTT)::
  - fully open / close button
  - slider open / close by xx%
How these functions work in Domoticz you need to ask on the Domoticz forum. Here you are unlikely to be helped.

**Reply by smitec11** [reply]
thanks for the reply. how did you setup the Tasmota side for it?

**Reply by TrDA-hab** [reply]
No additional settings are required on the Tasmota side.
